src/formatDate.js

```diff
@@ -50,7 +50,47 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let indexYearsTo;
+  let yearsToLength;
+
+  const separatorFrom = fromFormat[fromFormat.length - 1];
+
+  const dates = date.split(separatorFrom);
+
+  const formatedDate = Array(3);
```

Comment:
```suggestion
  const formatedDate = [];
```
так тоже будет работать

Reply:
якщо далі використовувати fill, то 
const formatedDate = [];
не працює

Reply:
done